frontend/src/app/libs/common-components/components/button/button.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.scss']
+})
+export class ButtonComponent {
+
+	@Input()
+	public type: string = 'basic';
+
+	@Input()
+	public isDark: boolean = false;
```

Comment:
let's call it `theme` and in consumes `ComponentTheme` enum (add it to /common-components/shared)